hack/generator/pkg/astmodel/known_locals_set.go

```diff
@@ -89,36 +114,13 @@ func (locals *KnownLocalsSet) HasName(name string) bool {
 
 // Clone clones the KnownLocalsSet
 func (locals *KnownLocalsSet) Clone() *KnownLocalsSet {
-	return &KnownLocalsSet{
-		idFactory: locals.idFactory,
-		names:     locals.names.clone(),
+	names := make(map[string]struct{}, len(locals.names))
```

Comment:
We should probably have a `StringSet` type since `map[string]struct{}` appears a bunch of times.

Reply:
I'll add that to a CodeGardening TODO list that I have.